api/tools/src/main/resources/data_dictionary_exports/AllofUsDatasetV4_20201207.yaml

```diff
@@ -0,0 +1,7920 @@
+meta_data:
+  -
+    id: '1dsvJV8B7EXQj5EWa2XG-KAhs-l7FsQnyJSSFMstLF2U'
+    name: 'All of Us Registered Tier CDR Data Dictionary'
+    version: 3034
+    created_time: 2020-12-07 14:17:00
```

Comment:
I think this created_time needs to be updated. The data dictionary will only load new entries if the created_date is newer than the version it currently knows.

Reply:
done